noxfile.py

```diff
@@ -75,7 +75,7 @@ def lint(session):
     session.run("python", "utils/license-headers.py", "check", *SOURCE_FILES)
     session.run("black", "--check", "--target-version=py38", *SOURCE_FILES)
     session.run("isort", "--check", "--profile=black", *SOURCE_FILES)
-    session.run("flake8", "--ignore=E501,W503,E402,E712,E203", *SOURCE_FILES)
+    session.run("flake8", "--extend-ignore=E203,E402,E501,E704,E712", *SOURCE_FILES)
```

Comment:
Using `--extend-ignore` is cleaner and recommended here: https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#flake8. E704 is triggered by a new black rule, and we did not need W503.